### ISSUE TITLE
About line and image position

### ISSUE THREAD

**Issue by leon2835** [codesinghanoop/react-native-d3-tree-graph#7]
Hi there, is there anyway to make the line align to the center of image, and the image become infront of the line( so that wont show the line above the picture? Thank You
![Screenshot_1564718413](https://user-images.githubusercontent.com/19604759/62343434-7863db80-b51d-11e9-9e89-21e5803e786a.png)

**Comment by codesinghanoop** [comment]
@leon2835 yeah, it is possible to need to resolve the issue. If you have then kindly contribute. Thanks will get back to you soon.
